src/assets/translations/fr/main.json

```diff
@@ -1308,7 +1317,7 @@
       "receiveAsset": "Recevoir des %{asset}",
       "onlySend": "Seul %{asset} (%{symbol}) peut être envoyé à cette adresse. L'envoi de toute autre monnaie peut entraîner une perte définitive.",
       "copy": "Copier",
-      "copied": "Adresse de réception pour %{symbol} copiée vers le presse-papier",
+      "copied": "Adresse de réception pour %{symbol} copiée vers le presse-papiers",
```

Comment:
or I guess that works too with plural 🤯 
> En [informatique](https://fr.wikipedia.org/wiki/Informatique), un presse-papier (ou presse-papiers selon l'orthographe toujours utilisée dans les dernières versions de [Windows](https://fr.wikipedia.org/wiki/Microsoft_Windows)[1](https://fr.wikipedia.org/wiki/Presse-papiers_(informatique)#cite_note-1) et [macOS](https://fr.wikipedia.org/wiki/MacOS)[2](https://fr.wikipedia.org/wiki/Presse-papiers_(informatique)#cite_note-2))
```suggestion
      "copied": "Adresse de réception pour %{symbol} copiée vers le presse-papier",
```

Reply:
I've actually changed this because it wasn't consistent with the other occurrences... So if you insist on the singular we'd have to update the other ones too ;)

Reply:
Totally agnostic!

Reply:
Then let's keep it plural, it will contrast with "porte-monnaie" which is invariable despite the fact we have almost infinite kinds of "monnaies" in crypto wallets... You have to love French sometimes 😅